packages/didier-stevens-suite.vm/tools/chocolateyinstall.ps1

```diff
@@ -17,10 +17,12 @@ try {
     $toolDir = Get-Item "${Env:RAW_TOOLS_DIR}\DidierStevensSuite-*"
     VM-Assert-Path $toolDir
 
-    # Add shortcut for commonly used PDF tools
+    # Add shortcut for commonly used python PDF tools
     ForEach ($toolName in @('pdfid', 'pdf-parser')) {
-      $executablePath = Join-Path $toolDir "$toolName.py"
-      VM-Install-Shortcut $toolName $category $executablePath -consoleApp $true -arguments "--help"
+      $executablePath = (Get-Command python).Source
```

Comment:
could we use python only as icon (now that we have this argument) instead of changing how we are executing the tool? 🤔

Reply:
The reason I did this is because it makes the shortcut more useful. 🙂 Without the change, I get asked how I want to open the file (though this is likely because I have VSCode also installed, which seems to make things difficult).

Without the change but with the icon being Python (your suggestion, but with VSCode installed):
![image](https://github.com/mandiant/VM-Packages/assets/35854880/c8c86e25-24da-43e9-9357-ce3119195b41)


With the change I made (even with VSCode installed):
![image](https://github.com/mandiant/VM-Packages/assets/35854880/03b62a37-1eee-439d-9cc1-4c954499ae3b)

Reply:
That is a good reason to specify the application to run it with. @mandiant/flare-vm any objections?